stratum/lib/security/credentials_manager.h

```diff
@@ -28,10 +28,19 @@ class CredentialsManager {
   virtual std::shared_ptr<::grpc::ServerCredentials>
   GenerateExternalFacingServerCredentials() const;
 
-  // Loads new credentials.
-  virtual ::util::Status LoadNewCredential(const std::string& ca_cert,
-                                           const std::string& cert,
-                                           const std::string& key);
+  // Generates client credentials for contacting an external facing gRPC server.
+  virtual std::shared_ptr<::grpc::ChannelCredentials>
+  GenerateExternalFacingClientCredentials() const;
```

Comment:
Maybe we should change the `Generate` to `Get` in the future since we actually generate client/server credentials when initializing the manager.